common-content/en/module/js1/arrow-functions/index.md

```diff
@@ -0,0 +1,60 @@
++++
+title = '➡️ Arrow functions'
+
+time = 5
+facilitation = false
+emoji= '➡️'
+[objectives]
+    1='Write an arrow function'
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false
+
++++
+
+As we write more code, we are going to write lots and lots of anonymous functions.
+
+JavaScript added even shorter ways of writing an anonymous function. These four functions all do the same thing:
+
+```js
+function convertToPercentage(decimalNumber) {
+  return `${decimalNumber * 100}%`;
+}
+```
+
+```js
+// We can skip the name of the function if we don't need it to have a name.
+function (decimalNumber) {
+  return `${decimalNumber * 100}%`;
+}
+```
+
+```js
+// We can also skip the keyword function.
+// If we do this, we need an arrow between our parameters and the function body.
+(decimalNumber) => {
+  return `${decimalNumber * 100}%`;
+}
+```
+
+```js
+// If our function just returns a single value,
+// without needing any other statements in our function,
+// we can even skip the return keyword.
+(decimalNumber) => `${decimalNumber * 100}%`;
+```
+
+This can be useful to make is quicker and easier to write functions. It also reduces the number of things we need to skip past when reading a function.
```

Comment:
```suggestion
This can make it easier and quicker to write functions. It reduces the number of things we need to read in a function. 
```

Reply:
I'm keeping the also unless you're going to fight me over it - reading and writing are different concerns, and making clear the second sentence is adding to, not clarifying, the first feels useful.